Trappist/src/Promact.Trappist.Web/wwwroot/app/reports/test-report/test-report.component.ts

```diff
@@ -4,10 +4,14 @@ import { Router, ActivatedRoute } from '@angular/router';
 import { TestAttendee } from '../testAttendee';
 import { TestStatus } from '../enum-test-state';
 import { Test } from '../../tests/tests.model';
-import * as Excel from 'exceljs/dist/exceljs.min.js';
+import { ConductService } from '../../conduct/conduct.service';
+import { TestInstructions } from '../../conduct/testInstructions.model';
+import { ReportQuestionsCount } from './reportquestionscount';
+import { TestAttendeeRank } from './testattendeerank';
 
 declare let jsPDF: any;
 declare let saveAs: any;
+let Excel = require('exceljs');
```

Comment:
keep it as it is in previous commit (import * as Excel from 'exceljs/dist/exceljs.min.js')